variables.tf

```diff
@@ -25,3 +25,9 @@ variable "lock_table_read_capacity" {
 variable "lock_table_write_capacity" {
   default = 1
 }
+
+variable "kms_key_id" {
+  # Default to absent/blank to use the default aws/s3 aws kms master key
```

Comment:
Ok, so this comment is no longer true, I think.

If the user wants to use the default aws/s3 AWS KMS master key then they must explicitly set the `kms_key_id` value to `alias/aws/s3`. Is that true?

Assuming that is true, must they also explicitly set the `kms_key_id` value in their [terraform backend configuration block](https://www.terraform.io/docs/backends/types/s3.html#kms_key_id) to `alias/aws/s3`?

Reply:
No, terraform will automatically use the default account master kms key for you. This comment holds true